src/swagger-2.ts

```diff
@@ -22,7 +22,7 @@ export interface Swagger2 {
 
 export interface Swagger2Options {
   camelcase?: boolean;
-  wrapper?: string;
+  wrapper?: string | false;
```

Comment:
Rather than having this be a union type, maybe we should just have `const shouldUseWrapper = options.wrapper !== undefined` ? And we can just make this property optional.

Reply:
Ah sorry—brainfart. It is nice having `declare namespace OpenAPI` be the default. I’m fine with this as-is.